api/handlers.go

```diff
@@ -1200,22 +1200,14 @@ func (api *API) trackDownloadPOST(u *database.User, w http.ResponseWriter, req *
 }
 
 // trackRegistryReadPOST registers a new registry read in the system.
-func (api *API) trackRegistryReadPOST(u *database.User, w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
-	_, err := api.staticDB.RegistryReadCreate(req.Context(), *u)
-	if err != nil {
-		api.WriteError(w, err, http.StatusInternalServerError)
-		return
-	}
+func (api *API) trackRegistryReadPOST(_ *database.User, w http.ResponseWriter, _ *http.Request, _ httprouter.Params) {
+	// Tracking registry reads is disabled.
 	api.WriteSuccess(w)
```

Comment:
nit: shouldn't we return an error if it is disabled?

Reply:
We could. I don't expect nginx to do anything about it, though. It will only log an error, which is going to fill our error logs while providing no actionable information.

I don't feel strongly about this, so I'm fine with adding an error if you (or anyone else) prefer.